lib/chef/provider/aws_vpc.rb

```diff
@@ -243,7 +244,9 @@ def update_internet_gateway(vpc)
         elsif current_ig != new_ig
           Cheffish.inline_resource(self, action) do
             aws_internet_gateway current_ig do
-              if current_ig.tags['OwnedByVPC'] == vpc.id
+              ig_tag = current_ig.tags.find{|i|i.key=='OwnedByVPC'}
```

Comment:
[Style] need space between `find` and `{`  and each side of  `==` like my above comment.